package-lock.json

```diff
@@ -1,14 +1,1222 @@
 {
```

Comment:
Why is this file now committed? I cannot see any reference to it, or its explanation. That needs to be included if this is to remain.

Reply:
It's not a new file. I'm not sure why it got so much larger but as `lockfileVersion` was bumped that's probably the reason.

Reply:
The file is managed by npm so the change was automatic.